packages/dd-trace/test/appsec/iast/analyzers/ldap-injection-analyzer.ldapjs.plugin.spec.js

```diff
@@ -6,12 +6,17 @@ const iastContextFunctions = require('../../../../src/appsec/iast/iast-context')
 const { newTaintedString } = require('../../../../src/appsec/iast/taint-tracking/operations')
 const vulnerabilityReporter = require('../../../../src/appsec/iast/vulnerability-reporter')
 const agent = require('../../../plugins/agent')
+const semver = require('semver')
 
 const base = 'dc=example,dc=org'
 
+const isOldNode = semver.satisfies(process.version, '<=14')
+
 describe('ldap-injection-analyzer with ldapjs', () => {
   let client
   withVersions('ldapjs', 'ldapjs', version => {
+    if (isOldNode && version !== '2.0.0') return
```

Comment:
Should the `version` check here not _also_ be a `semver.satisfies(...)` or will this definitely always match? Wouldn't a new release break that check? Or am I misunderstanding?

Reply:
The idea is to test only 2.0.0 version with old node versions because newer releases like version 3.0.0 are using `String.replaceAll`, available since node v15.

Reply:
Until now only two versions: 2.0.0 and 2.3.3 were tested but with the release of 3.0.0, the test runs with 2.0.0 and 3.0.0 versions. So, I assume 2.0.0 will be always tested even if a new version is released.